build/crds/build-crds.sh

```diff
@@ -51,6 +51,21 @@ copy_ob_obc_crds() {
   cp -f "${SCRIPT_ROOT}/deploy/olm/assemble/objectbucket.io_objectbuckets.yaml" "$OLM_CATALOG_DIR"
 }
 
+copy_volume_replication_crds() {
+  volume_replication_dir=${SCRIPT_ROOT}/.cache/crds/volumereplication
+  volume_replication_crd=replication.storage.openshift.io_volumereplications.yaml
+  volume_replication_class_crd=replication.storage.openshift.io_volumereplicationclasses.yaml
+  if [[ ! -d "${volume_replication_dir}" ]]; then
+    mkdir -p "${volume_replication_dir}"
+    VOLUME_REPLICATION_URL=${VOLUME_REPLICATION_URL:-https://raw.githubusercontent.com/csi-addons/volume-replication-operator/v0.3.0/config/crd/bases}
```

Comment:
@jarrpa The url (including both the fork and version) can be overridden by setting the env var `VOLUME_REPLICATION_URL`